src/functions/leaderboard/leaderboard.go

```diff
@@ -0,0 +1,27 @@
+package leaderboard
+
+import (
+	"strings"
+	"encoding/json"
+	"net/http"
+)
+
+const BASE_URL = "https://archybot.web.app/leaderboard/"
+
+var Payload struct {
```

Comment:
Change `var` for `type`
That's my mistake in the `go` command

Reply:
Ok! I thought that it was because it must be a var instead of type. I will correct it!